packages/embeds/embed-core/playground.ts

```diff
@@ -474,7 +474,7 @@ if (only === "all" || only == "ns:monthView") {
     //@ts-ignore
     {
       elementOrSelector: "#cal-booking-place-monthView .place",
-      calLink: "pro/paid",
+      calLink: "free/30min",
```

Comment:
Paid didn't have availability on my local due to some reason, so changed it to free/30min as it is testing layout only